src/components/RangeInput.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+<div v-if="state" :class="suit()">
+  <form :class="suit('form')" @submit.prevent="refine()">
```

Comment:
We should have a slot to support customization

Reply:
A global one?

Reply:
Yes a default one rather than global depends the definition of global in your question. You can take a look at other [widget implementation](https://github.com/algolia/vue-instantsearch/blob/bb011b70599b3fc1b72b225489d49e1c8e761dbe/src/components/SearchBox.vue#L3-L6).